src/geff/_cli.py

```diff
@@ -1,31 +1,159 @@
 import logging
 import sys
+from pathlib import Path
+from typing import TYPE_CHECKING, Annotated, cast
 
 import typer
 import zarr
 
 from . import utils
 from .metadata_schema import GeffMetadata
 
+if TYPE_CHECKING:
+    from typing import Literal
+
 app = typer.Typer(help="GEFF Command Line Interface")
 
 logging.basicConfig(stream=sys.stderr, level=logging.WARNING, format="%(levelname)s: %(message)s")
 logging.captureWarnings(True)
 
 
 @app.command()
-def validate(input_path: str = typer.Argument(..., help="Path to the GEFF file")):
+def validate(
+    input_path: str = typer.Argument(
+        ..., help="Path to the GEFF folder, e.g. data.zarr/tracks.geff"
+    ),
+):
     """Validate a GEFF file."""
     utils.validate(input_path)
     print(f"{input_path} is valid")
 
 
 @app.command()
-def info(input_path: str = typer.Argument(..., help="Path to the GEFF file")):
+def info(
+    input_path: str = typer.Argument(
+        ..., help="Path to the GEFF folder, e.g. data.zarr/tracks.geff"
+    ),
+):
     """Display information about a GEFF file."""
     metadata = GeffMetadata.read(zarr.open(input_path, mode="r"))
     print(metadata.model_dump_json(indent=2))
 
 
+@app.command()
+def convert_ctc(
```

Comment:
it would be nifty (if perhaps require a bit more "magic") to have one `geff convert` ... rather than a different `convert_x` for each thing you intend to support?  seems to me like it should be possible to have a `guess_format(path)` helper function that takes the input and infers the format?

on the other hand... if you know that each conversion format is going to require a lot of format-specific arguments, maybe not...

Reply:
I'm game to try to have a general convert that just uses all defaults if that makes sense. Need to think about it a bit more closely